packages/logger/tests/unit/logBuffer.test.ts

```diff
@@ -0,0 +1,123 @@
+import { describe, expect, it, vi } from 'vitest';
+import { CircularMap, SizedItem, SizedSet } from '../../src/logBuffer';
+
+describe('SizedItem', () => {
+  it('should calculate the byteSize based on JSON serialization', () => {
+    // Prepare
+    const logEntry = { message: 'hello world' };
```

Comment:
Let's update all the tests to use strings, basically we run `JSON.stringify()` on the log item, before adding it.